sql/sys_vars.cc

```diff
@@ -5408,12 +5408,6 @@ static Sys_var_ulong Sys_wsrep_mysql_replication_bundle(
        GLOBAL_VAR(wsrep_mysql_replication_bundle), CMD_LINE(REQUIRED_ARG),
        VALID_RANGE(0, 1000), DEFAULT(0), BLOCK_SIZE(1));
 
-static Sys_var_mybool Sys_wsrep_load_data_splitting(
-       "wsrep_load_data_splitting", "To commit LOAD DATA "
-       "transaction after every 10K rows inserted",
-       GLOBAL_VAR(wsrep_load_data_splitting), 
-       CMD_LINE(OPT_ARG), DEFAULT(TRUE));
-
```

Comment:
This appears to be important for users of Galera replication. This parameter is a work-around for Galera, which cannot handle large transactions.
I think that we must keep this parameter, but move the related logic (which was removed from `ha_innobase::write_row()`) to the implementation of `LOAD DATA`, that is, `mysql_load()`. Could you please try to do that?

Reply:
Ok，I will do it. Thank you Marko.

Reply:
Thank you Xianliang, I am looking forward to it.